python/lsst/ap/association/skyBotEphemerisQuery.py

```diff
@@ -231,10 +233,12 @@ def _skybotConeSearch(self, expCenter, epochJD, queryRadius):
             columns = [col.strip() for col in dfSSO.columns]
             coldict = dict(zip(dfSSO.columns, columns))
             dfSSO.rename(columns=coldict, inplace=True)
-            # TODO: DM-31934 Replace custom conversion code with
-            #       Astropy.coordinates.
-            dfSSO["ra"] = self._rahms2radeg(dfSSO["RA(h)"])
-            dfSSO["decl"] = self._decdms2decdeg(dfSSO["DE(deg)"])
+            # Data returned in hourangle format.
+            radec = dfSSO["RA(h)"]+' '+dfSSO["DE(deg)"]
+            skycoo = SkyCoord(radec.values, unit=(u.hourangle, u.deg))
```

Comment:
Isn't it a bit overkill to use `SkyCoord` here?

```python
from astropy.coordinates import Angle

dfSSO["ra"] = Angle(dfSSO["RA(h)"])
dfSSO["decl"] = Angle(dfSSO["DE(deg)", unit="deg")
```
eg.:
```python
>>> c.Angle("5 32 3.4", unit="deg").deg
5.534277777777778
>>> c.Angle("5 32 3.4", unit="h").deg
83.01416666666665
```

Reply:
I would have used `Angle` myself, yes, but the most important thing is to have an off-the-shelf solution.

Reply:
SkyCoord adds an extra layer of indirection and requires you to concatenate the two strings. I agree that anything is better than doing it ourselves.

Reply:
The optimization here seems pretty tiny--we're going to throw this whole module away--but sure, that works.